README.md

```diff
@@ -19,7 +19,7 @@ There are two ways to configure `ember-cli-bugsnag`:
 {
```

Comment:
I plan to bump the major version to 3.0, can you add a section to the readme that mentions 3.0 contains breaking changes?